src/pkg/logger/logger.go

```diff
@@ -112,9 +133,10 @@ func addFlags(fs *pflag.FlagSet, defaultFile string) {
 
 // Settings records the user's preferred logging settings.
 type Settings struct {
-	File        string // what file to log to (alt: stderr, stdout)
-	Level       string // what level to log at
-	PIIHandling string // how to obscure pii
+	File        string    // what file to log to (alt: stderr, stdout)
+	Format      logFormat // whether to format as text (console) or json (cloud)
+	Level       logLevel  // what level to log at
```

Comment:
these are now both strings and can be passed in, uninitialized via `Seed`. What happens if they're never initialized?

Reply:
Yeah, that's all getting a bit cluttered, isn't it?  I did a cleanup here: https://github.com/alcionai/corso/pull/3401.  We can squash that into this branch when its ready.